### ISSUE TITLE
"Save failed" (dashboards) when adding multiple questions with multiple date filters

### ISSUE THREAD

**Issue by NicoBerthoz** [metabase/metabase#9375]
Hi all,
I didn't find this exact problem amongst other issues.

### Bugs
If you're filing an issue about a bug please include as much information
as you can including the following.

- Your browser and the version: (e.x. Chrome 52.1, Firefox 48.0, IE 10)

- Your operating system: (e.x. OS X 10, Windows XP, etc)
Mac OS 10.14.2
Using Chrome - Version 71.0.3578.98 (Build officiel) (64 bits) 
- Your databases: (e.x. MySQL, Postgres, MongoDB, …)
MySQL
- Metabase version: (e.x. 0.19.3)
.31.2
- Metabase hosting environment: (e.x. Mac app, Elastic Beanstalk, Docker, Heroku, Linux/Ubuntu 12)
AWS
- Metabase internal database: (e.x. H2 (default), MySQL, Postgres)
Not sure how i could find that, i'd say MySQL

- *Repeatable steps to reproduce the issue*
I create two graphs 1/2 (number of deals, volume of deals (€)) with a time filter (all options)
The query looks like this:

```
FROM `deals`
WHERE {{date_range}}

```

I duplicate these graphes (version A/B). 
- Version A is for the total volume, 
- B is for the split between new & returning customers.

I don't manage to create 1A/1B & 2A/2B together on the same graph.
It's actually weird because sometimes it will fail when i add 1B, sometimes it will fail when I add 2A/2B, sometimes it will fail when i add 1B/2B. I tried all the combinations and it always fails.
Example of errors:

```
app-main.bundle.js?78262faa1c529a733077:5 {status: 500, data: {…}, isCancelled: false}data: {value: Array(1), error: Array(1), message: "Input to field-clause->id-or-literal does not matc…:expression, :binning-strategy") clause)] [0m ↵↵", type: "class clojure.lang.ExceptionInfo", stacktrace: Array(32)}isCancelled: falsestatus: 500__proto__: Object
(anonymous) @ app-main.bundle.js?78262faa1c529a733077:5
Promise.then (async)
n.onClick @ app-main.bundle.js?78262faa1c529a733077:5
i @ vendor.bundle.js?78262faa1c529a733077:96
s @ vendor.bundle.js?78262faa1c529a733077:96
executeDispatchesInOrder @ vendor.bundle.js?78262faa1c529a733077:96
h @ vendor.bundle.js?78262faa1c529a733077:94
f @ vendor.bundle.js?78262faa1c529a733077:94
t.exports @ vendor.bundle.js?78262faa1c529a733077:139
processEventQueue @ vendor.bundle.js?78262faa1c529a733077:94
(anonymous) @ vendor.bundle.js?78262faa1c529a733077:145
handleTopLevel @ vendor.bundle.js?78262faa1c529a733077:145
f @ vendor.bundle.js?78262faa1c529a733077:145
perform @ vendor.bundle.js?78262faa1c529a733077:94
batchedUpdates @ vendor.bundle.js?78262faa1c529a733077:145
batchedUpdates @ vendor.bundle.js?78262faa1c529a733077:83
dispatchEvent @ vendor.bundle.js?78262faa1c529a733077:145
```
and
`VM339:1 POST https://metabase.fairmoney.com.ng/api/dashboard/16/cards 500`

Thanks in advance for your help, would love to go through that one!

**Comment by flamber** [comment]
Pretty sure this is a duplicate of #8917 which is fixed in the upcoming 0.32.0 release.

**Comment by NicoBerthoz** [comment]
My bad then, i had seen the other issue but thought it was a different usecase. I'll close & will open again if different. thanks!
